create_deposition/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+curl --header "Content-Type: application/json" \
+      --request POST \
+      --data @$ZENODO_METADATA \
+      https://sandbox.zenodo.org/api/deposit/depositions?access_token=$ZENODO_API_TOKEN \
+      -o ./create_deposition/resp.json
```

Comment:
the action runs within a Docker container. Per the github actions spec, the output folder that is visible within the container is the one associated to the `GITHUB_WORKSPACE` variable, so we should write to `$GITHUB_WORKSPACE/create_response.json` instead.